src/patternfly/components/Card/card-sr-input.hbs

```diff
@@ -0,0 +1,8 @@
+<input
+  type="checkbox"
+  tabindex="-1"
+  class="pf-c-card__sr-input pf-screen-reader
+    {{#if card-sr-input--modifier}}
+      {{card-sr-input--modifier}}
+    {{/if}}"
+/>
```

Comment:
nit - this would align more with how a standard hbs element template is setup. You could put the modifier stuff on it's own lines if you want, just use the `~` to remove the leading whitespace, like `{{~#if card-sr-input--modifier}}`

also in core, html5 doesn't require the `/` on self closing elements

```suggestion
<input class="pf-c-card__sr-input pf-screen-reader{{#if card-sr-input--modifier}} {{card-sr-input--modifier}}{{/if}}"
  type="checkbox"
  tabindex="-1"
  {{#if card-sr-input--attribute}}
    {{{card-sr-input--attribute}}}
  {{/if}}>
```